fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -409,15 +409,23 @@ case $DistortionCorrection in
         Files="PhaseOne_gdc_dc PhaseTwo_gdc_dc SBRef_dc"
         for File in ${Files}
         do
+            # check if T2w image exists
+            if [ `${SubjectFolder}/T1w/T2w_acpc_dc.nii.gz` -eq 0 ]; then
```

Comment:
Backticks do command substitution, and the image file, whether or not it exists, is not an executable.  You want double quotes, `""`.

Also, `-eq 0` is not how you test for file existence (it should error, as the string does not parse as an integer).  You want `-f <file>`.

Reply:
Oh, I see - you forgot to put `imgtest` in the backticks.

Reply:
Yes. I changed the code we used to have to follow @mharms guidance and the imtest fell out during copy/paste. Corrected now.